### DISCUSSION TITLE
About Nuxt

### DISCUSSION THREAD

**Discussion by sphinxDevVic** [sphinxDevVic/vuex#4] [Ideas]
Please discuss about Nuxt and Quasar framework

**Comment by taeul** [comment]
Hello sphinxDevVic

It is very excited and I want to work with you.

Thank you.

**Reply by sphinxDevVic** [reply]
Thanks. Nice to meet you!!

**Comment by taeul** [comment]
Okay , I want to work with you

**Comment by sphinxDevVic** [comment]
Thanks
